README.md

```diff
@@ -1,30 +1,57 @@
-For each of them, it will be possible to see the included branch and change it to another existing branch.  The selected branch can be updated by performing a pull operation. 
-The next option is to select a mod and build the project according to it.
-And lastly launching the project. Also these possibilities should be available for a group of projects, i.e. the selection of settings and then the execution of the necessary logic for all of them.
+The tool should be able to:
+
+1. Synchronize repositories (all or selected). Pull the branch that is currently selected. Selected branch shall be visible to user.
+2. Have the ability to change between schemas - change multiple repositories to 9.0 or 10.0 (master). Synchronization is considered a separate operation.
+3. Have an autostash feature - be able to stash changes for current branch before pulling.
+4. Have a yaml configuration file in a repo to get the configuration from.
+5. Automatically prune branches that do not have remote branches.
+6. Have the ability to manually add repositories for synchronization from FossConnect project.
+7. Have some short meaningfull output in the UI and detailed logs in a file. There should be a dedicated 'View logs' button to open log folder for convinience.
+8. Introduce profiles that will contain list of repositories that should be checked for synchronization.
+9. Build application
+10. Run application
 
 ### Implementation
 
-This will be a desktop application developed using WPF. The configuration file mentioned earlier will be local in the first iteration, in the future it is planned to load it from a remote repository. 
-It is planned to use the git application to work with repositories because the pool of scheduled tasks does not require complex commands. List of commands that are planned to be used are : branch, checkout, pull, stash. 
+This will be a desktop application developed using WPF targeting .NET 6. It is planned to use the Git CMD application to work with repositories because the pool of scheduled tasks does not require complex commands. List of commands that are planned to be used are : branch, checkout, pull, stash.
+
+The configuration file mentioned earlier will be local in the first iteration, in the future it is planned to load it from a remote repository. 
+
 All information will be presented in a table. One line in it corresponds to one project. 
-To select the necessary branch there will be a ComboBox with an available selection. After branch selection, SourceExplorer will check if the branch is presented locally. If not - it will perform a pull operation. 
-Also, there will be a ComboBox for build mode.
-For each operation, buttons are provided to perform it. If action is required to be performed on multiple projects, user must click on corresponding CheckBoxes and then on an action button.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/fulltable.png)
+
```

Comment:
We need a border and a top margin here:
![image](https://user-images.githubusercontent.com/42497175/143225139-859b9bb0-3196-4511-92db-bc9d634b158f.png)

Reply:
delete resize

Reply:
window can be resize this size we have to save

Reply:
fix resize